dotnet/hello_world/Program.cs

```diff
@@ -0,0 +1,210 @@
+                    }
+                    else
+                    {
+                        Console.WriteLine($"Table {_settings.TableName} already exist");
+                    }
+
+                    #endregion
+
+                    #region Insert multiple rows in bulk using MutateRows
+
+                    /* Each row has a unique row key.
+                                       
+                       Note: This example uses sequential numeric IDs for simplicity, but
+                       this can result in poor performance in a production application.
+                       Since rows are stored in sorted order by key, sequential keys can
+                       result in poor distribution of operations across nodes.
+                     
+                       For more information about how to design a Bigtable schema for the
+                       best performance, see the documentation:
+                      
+                       https://cloud.google.com/bigtable/docs/schema-design */
+
+                    Console.WriteLine($"Writing some greetings to the table {_settings.TableName}");
+
+                    //string[] greetings = new string[] { "Hello World!", "Hellow Bigtable!", "Hellow C#!" };
+
+                    BigtableClient bigtableClient = BigtableClient.Create();
+
+                    //building a MutateRowsRequest (contains table name and collection of entries)
+                    MutateRowsRequest request = new MutateRowsRequest
+                    {
+                        TableNameAsTableName = _tableClient
+                    };
+
+                    for (int i = 0; i < _settings.Greetings.Length; i++)
+                    {
+                        string rowKey = String.Format("{0}{1}", _settings.RowKeyPrefix, i);
+
+                        //Building an entry for every greeting (contains of row key and collection of mutations)
+                        MutateRowsRequest.Types.Entry _entry = new Google.Cloud.Bigtable.V2.MutateRowsRequest.Types.Entry();
+                        _entry.Mutations.Add(Mutations.SetCell(_settings.ColumnFamily, _settings.ColumnName, new BigtableByteString(_settings.Greetings[i])));
+                        _entry.RowKey = ByteString.CopyFromUtf8(rowKey);
+
+                        //adding entry to request
+                        request.Entries.Add(_entry);
+                    }
+
+                    BigtableClient.MutateRowsStream response = bigtableClient.MutateRows(request);
+
+                    Task write = CheckWrittenAsync();
+                    write.Wait();
+
+                    async Task CheckWrittenAsync()
+                    {
+                        IAsyncEnumerator<MutateRowsResponse> responseStream = response.ResponseStream;
+                        while (await responseStream.MoveNext())
+                        {
+                            MutateRowsResponse current = responseStream.Current;
```

Comment:
This variable is unused?

Reply:
This variable can be used to check whether rows where written successfully:
    
```
MutateRowsResponse current = responseStream.Current;

// Check whether rows where written successfully
foreach (var entry in current.Entries) {
       if(entry.Status.Code == 0) {
            Console.WriteLine($"\tRow key: " +
                    $"{request.Entries[(int)entry.Index].RowKey.ToStringUtf8()} written successfully");
       } else {
            Console.WriteLine($"\tFailed to write Row key: " +
                    $"{request.Entries[(int)entry.Index].RowKey.ToStringUtf8()}");
            Console.WriteLine(entry.Status.Message);
       }
}
```